adafruit_bmp3xx.py

```diff
@@ -57,80 +65,78 @@
+            raise RuntimeError(f"Failed to find BMP3XX! Chip ID {hex(chip_id)}")
         self._read_coefficients()
         self.reset()
         self.sea_level_pressure = 1013.25
         self._wait_time = 0.002  # change this value to have faster reads if needed
         """Sea level pressure in hPa."""
 
     @property
-    def pressure(self):
+    def pressure(self) -> float:
         """The pressure in hPa."""
         return self._read()[0] / 100
 
     @property
-    def temperature(self):
+    def temperature(self) -> float:
         """The temperature in degrees Celsius"""
         return self._read()[1]
 
     @property
-    def altitude(self):
+    def altitude(self) -> float:
         """The altitude in meters based on the currently set sea level pressure."""
         # see https://www.weather.gov/media/epz/wxcalc/pressureAltitude.pdf
         return 44307.7 * (1 - (self.pressure / self.sea_level_pressure) ** 0.190284)
 
     @property
-    def pressure_oversampling(self):
+    def pressure_oversampling(self) -> int:
         """The pressure oversampling setting."""
         return _OSR_SETTINGS[self._read_byte(_REGISTER_OSR) & 0x07]
 
     @pressure_oversampling.setter
-    def pressure_oversampling(self, oversample):
+    def pressure_oversampling(self, oversample: Literal[1, 2, 4, 8, 16, 32]) -> None:
         if oversample not in _OSR_SETTINGS:
-            raise ValueError("Oversampling must be one of: {}".format(_OSR_SETTINGS))
+            raise ValueError(f"Oversampling must be one of: {_OSR_SETTINGS}")
         new_setting = self._read_byte(_REGISTER_OSR) & 0xF8 | _OSR_SETTINGS.index(
             oversample
         )
         self._write_register_byte(_REGISTER_OSR, new_setting)
 
     @property
-    def temperature_oversampling(self):
+    def temperature_oversampling(self) -> int:
         """The temperature oversampling setting."""
         return _OSR_SETTINGS[self._read_byte(_REGISTER_OSR) >> 3 & 0x07]
 
     @temperature_oversampling.setter
-    def temperature_oversampling(self, oversample):
+    def temperature_oversampling(self, oversample: Literal[1, 2, 4, 8, 16, 32]) -> None:
         if oversample not in _OSR_SETTINGS:
-            raise ValueError("Oversampling must be one of: {}".format(_OSR_SETTINGS))
+            raise ValueError(f"Oversampling must be one of: {_OSR_SETTINGS}")
         new_setting = (
             self._read_byte(_REGISTER_OSR) & 0xC7 | _OSR_SETTINGS.index(oversample) << 3
         )
         self._write_register_byte(_REGISTER_OSR, new_setting)
 
     @property
-    def filter_coefficient(self):
+    def filter_coefficient(self) -> int:
         """The IIR filter coefficient."""
         return _IIR_SETTINGS[self._read_byte(_REGISTER_CONFIG) >> 1 & 0x07]
 
     @filter_coefficient.setter
-    def filter_coefficient(self, coef):
+    def filter_coefficient(self, coef: Literal[0, 2, 4, 8, 16, 32, 64, 128]) -> None:
```

Comment:
Same thing, can you pick either this type annotation or `int` and make the getter and setter match?